exporter/elasticexporter/README.md

```diff
@@ -1,5 +1,11 @@
 # Elastic native support for OpenTelemetry
 
+| Status                   |                  |
+| ------------------------ |------------------|
+| Stability                | [beta]           |
```

Comment:
```suggestion
| Stability                | [deprecated]           |
```

Since the Elastic 7.13 release, Elastic APM Server has supported OTLP directly. This `elastic` exporter was deprecated at that point. See https://www.elastic.co/guide/en/apm/guide/current/open-telemetry.html#open-telemetry-native